### PR TITLE
Allow Person#add_membership to warn instead of error out when no match

### PR DESCRIPTION
**Pull request by patcon**
Like how we handle `EventAgendaItem#add_bill`

### CODE REVIEW

**Comment by patcon** [comment]
Scenario where this would help us is for scraping councillor membership data when they list lots of unoffiicial orgs (BIA's, etc.) that we aren't [yet?] scraping into OCD.

Not sure if this is abusing use of error handling. Open to suggestions on how to do it better!

**Comment by patcon** [comment]
Grrrr... Can't get tests on this working for the life of me :/

**Comment by patcon** [comment]
Failing tests aside, any advice on whether this is a proper approach? If this doesn't feel right, then I'm not sure how to use `allow_no_match` when adding a membership. Any advice appreciated so that I can help move past this

cc: @jpmckinney

File: pupa/importers/memberships.py [hunk 1]
```diff
@@ -40,12 +40,16 @@ def prepare_for_db(self, data):
...
 
-        data['organization_id'] = self.org_importer.resolve_json_id(data['organization_id'])
+        data['organization_id'] = self.org_importer.resolve_json_id(data['organization_id'], allow_no_match=True)
         data['person_id'] = self.person_importer.resolve_json_id(data['person_id'])
         data['post_id'] = self.post_importer.resolve_json_id(data['post_id'])
         if not is_party:
             # track that we had a membership for this person
             self.seen_person_ids.add(data['person_id'])
+
+        if data['organization_id'] == None:
```
Comment (jpmckinney) [inline pupa/importers/memberships.py]: Just test `if not` or if you really need to test None then use `is None`.

**Comment by jpmckinney** [comment]
Why not just not scrape the memberships in orgs that don't exist?

**Comment by patcon** [comment]
I'm working around my immediate need for this PR, so it isn't too critical if you don't think it's useful. But this would be used in the people scraper, and the orgs are scraped in another one, so skipping them isn't something that's obvious to me how to do.

**Comment by jpmckinney** [comment]
Wouldn't it be simpler to control the order in which scrapers are run, so that orgs are always imported first before memberships are attempted to be imported?

**Comment by patcon** [comment]
Well, there's still be some orgs missing in this case. You're saying that if the `committee` scraper always ran first, then the `people` scraper would be able to "look up" non-existent committees and skip `add_membership` for those?

(I'm going to close this in the meantime, as I don't need it, and I suspect it requires a different approach to work nicely as well :)
